core/modules/config.go

```diff
@@ -90,6 +90,8 @@ type ModuleConfigUserFields struct {
 type SDK struct {
 	Source string         `json:"source"`
 	Config map[string]any `json:"config,omitempty"`
+	// The experimental features enabled for this module.
+	Experimental map[string]bool `json:"experimental,omitempty"`
```

Comment:
LGTM! (we've previously discussed separately)

Wondering if this should just be `[]string` though?

cc @shykes just checking this looks like the right place in `dagger.json` to have this setting?

Reply:
Yeah why not for `[]string`. The map has the advantage to directly query with the key instead of looping over the array, but as we don't expect a lot of keys, that might work too.